teku/src/main/java/tech/pegasys/teku/cli/subcommand/VoluntaryExitCommand.java

```diff
@@ -108,11 +107,14 @@ public void run() {
         confirmExits();
       }
       getValidatorIndices(validators).forEach(this::submitExitForValidator);
-    } catch (UncheckedIOException ex) {
+    } catch (InvalidConfigurationException ex) {
       if (Throwables.getRootCause(ex) instanceof ConnectException) {
         SUB_COMMAND_LOG.error(getFailedToConnectMessage());
         System.exit(1);
       }
```

Comment:
Could we add an `else` case here to rethrow the original exception if it isn't a `ConnectException`? Otherwise we'll silently ignore `InvalidConfigurationException` which could be quite confusing.

Reply:
Nice catch, fixed. My bad I enable auto-merge, let's open another one